resourcematch-app/package-lock.json

```diff
@@ -20,11 +21,13 @@
         "eslint-plugin-react-hooks": "^4.6.0",
         "eslint-plugin-react-refresh": "^0.4.3",
         "express": "^4.18.2",
+        "formik": "^2.4.5",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "sirv": "^2.0.3",
+        "styled-components": "^6.1.1",
         "ts-node": "^10.9.1",
-        "typescript": "^5.1.6",
+        "typescript": "^5.3.2",
```

Comment:
I think this change is from when we were trying to fix your VS Code issue yesterday. If you run `npm install` right now, it'll show a diff to `package-lock.json` :man_shrugging: .